utils/swift_build_support/swift_build_support/shell.py

```diff
@@ -80,40 +88,74 @@ def call(command, stderr=None, env=None, dry_run=None):
             "': " + e.strerror)
 
 
+def capture(command, stderr=None, env=None, dry_run=None, print_command=True):
```

Comment:
> For this method, print_command=False is relevant IMO.

Did you mean _irrelevant_? Or that the default should be different?

> BTW, Could you change print_command to echo or trace? it's easier to type.

I agree, I would prefer a shorter name too but I was preserving the existing name carried over from `SwiftBuildSupport`. If @gribozavr has no objections I will switch to `echo`.